target/linux/pistachio/base-files/etc/uci-defaults/02_enable_bt

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+ln -s /etc/init.d/bt-csr /etc/rc.d/S22bt-csr
+sleep 2
+/etc/rc.d/S22bt-csr start
```

Comment:
Is this workaround needed with the fix mentioned above? If so I would expect the file to look like this as it only runs on first boot:
```
#!/bin/sh

/etc/init.d/bt-csr enable
/etc/init.d/bt-csr start
sync
```